node_modules/col-activities/lib/notifications/weekly.js

```diff
@@ -93,6 +87,18 @@ var sendWeeklyNotificationsForCourse = module.exports.sendWeeklyNotificationsFor
  * @api private
  */
 var collectCourse = function(course, callback) {
+  // Do not send the email unless asset library and engagement index are both enabled
+  if (!course.assetlibrary_url || !course.engagementindex_url) {
+    log.info({'course': course}, 'Skipping weekly notification email for course without enabled tools');
+    return callback();
+  }
```

Comment:
This check was moved one method deeper so that it will catch both manually and automatically triggered emails.